### DISCUSSION TITLE
Budibase Support Contact

### DISCUSSION THREAD

**Discussion by ShivrajJoshi1996** [Budibase/budibase#9120] [Help]
Hi folks,

Does any one know how to stop budibase to autoupdate or can anyone please give me E-mail  Id of budibase support need it urgently.

Thanks

**Comment by melohagan** [comment]
Hey @ShivrajJoshi1996 

If you click on `Manage` beside your app then you can Revert to the previous version in the settings by clicking `Revert app`.
![Screenshot 2022-12-21 at 17 15 19](https://user-images.githubusercontent.com/101575380/208964930-5039a51a-13b8-419d-80b1-dc9c24b28e11.png)

![Screenshot 2022-12-21 at 17 15 35](https://user-images.githubusercontent.com/101575380/208964983-6f0a0711-8e7c-46a9-b4ac-f8179494ea97.png)
